python/dgl/transforms/module.py

```diff
@@ -368,39 +368,73 @@ class LaplacianPE(BaseTransform):
     <https://arxiv.org/abs/2003.00982>`__
 
     This module only works for homogeneous bidirected graphs.
-
+    
     Parameters
     ----------
     k : int
-        Number of smallest non-trivial eigenvectors to use for positional encoding
-        (smaller than the number of nodes).
+        Number of smallest non-trivial eigenvectors to use for positional encoding.
     feat_name : str, optional
         Name to store the computed positional encodings in ndata.
-
+    padding : bool
+        If padding=='false', raise exception when k>=n.
+        Else return (n-1) laplacian positional encodings and (k-n+1) zero encodings
+        (padding) when k>=n.
+        n is the number of nodes in the given graph.
+    return_eigval : bool
```

Comment:
How about use an optional argument `eigval_name` here instead? Set the default value to `None`, which represents the eigenvalues are not required. Otherwise, set the eigenvalue features in the field of `eigval_name` (e.g. g.ndata['eigval']). It may avoid possible name conflicts.